main.tf

```diff
@@ -1,15 +1,40 @@
+# This Terraform file defines the configuration for creating an IAM roles that allows CloudGuard access to the AWS account.
+
+# The `data "aws_partition" "current"` block retrieves information about the current AWS partition.
+data "aws_partition" "current" {}
+
+# The `data "aws_caller_identity" "current"` block retrieves information about the current AWS caller identity.
+data "aws_caller_identity" "current" {}
+
+# The `data "aws_region" "current"` block retrieves information about the current AWS region.
+data "aws_region" "current" {
+  lifecycle {
+    postcondition {
+      condition     = self.name == local.region
+      error_message = "Error: AWP must be deployed in the same region as the CloudGuard Data Center: ${local.region} (Not in ${self.name})"
+    }
+  }
+}
+
+data "aws_organizations_organization" "org" {}
```

Comment:
As discussed, can be removed?

Reply:
the variable is removed,
the data is used by the module and not required the user for any input